cooperator/security/ir_rule.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" ?>
+<odoo>
+    <record model="ir.rule" id="subscription_request_rule_company">
+        <field name="name">Subscription Request multi-company</field>
+        <field name="model_id" ref="cooperator.model_subscription_request" />
+        <field name="global" eval="True" />
+        <field name="domain_force">[('company_id', 'in', company_ids)]</field>
```

Comment:
There is a reason to not show the records to all companies, the ones that don't have any `company_id` assigned?

This is the standard in a multi-company logic, but maybe it doesn't make a lot of sense in  `cooperator`.